tket2/src/portmatching/pattern.rs

```diff
@@ -235,8 +236,14 @@ mod tests {
             edges,
             [
                 (cx_gate, h_gate),
-                (cx_gate, NodeID::CopyNode(inp, Port::new_outgoing(0))),
-                (cx_gate, NodeID::CopyNode(inp, Port::new_outgoing(1))),
+                (
+                    cx_gate,
+                    NodeID::CopyNode(inp, Port::new(Direction::Outgoing, 0))
```

Comment:
```suggestion
                    NodeID::CopyNode(inp, Port::new_outgoing(0))
```

Reply:
That doesn't exist anymore.

Reply:
Yes, I just found that myself, sorry. `OutgoingPort::from(0).into()`, hmmm. Add `NodeID::new_copy(Node, impl Into<Port>)` ?